.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+inherit_from:
+    - .rubocop_todo.yml
```

Comment:
Oh, I get it. I thought you mention `.rubocop.yml` file, but you mention `.rubocop_todo.yml` file